### ISSUE TITLE
How to contribute as a translator?

### ISSUE THREAD

**Issue by proneon267** [clearlinux/distribution#1480]
As a translator, I want to contribute to clear linux by translating it into my language. How do I get started?

**Comment by thiagomacieira** [comment]
Hello

Thank you for your interest in Clear! What part of it are you interested in translating?
1. Clear Linux website and documentation
1. Clear Linux software, like swupd, clr-boot-manager,r mixer, autospec etc.
1. Other software

For #3, we don't translate in Clear. Each software is responsible for providing up-to-date translations themselves. We can help you find their communities if you wish.

For the other two, we can definitely use the help!

**Comment by proneon267** [comment]
Yes, I would like to start by first translating the website and documentation and then Clear Linux Software. Where do I start?

**Comment by lebensterben** [comment]
>Yes, I would like to start by first translating the website and documentation and then Clear Linux Software. Where do I start?

@proneon267 There is a repository for documentation. 
https://github.com/clearlinux/clear-linux-documentation

You can fork the repository and translate the documentation, then make a pull request.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by proneon267** [comment]
Thanks! And what about the website?
